service/history/historyEngine.go

```diff
@@ -550,7 +550,9 @@ func (e *historyEngineImpl) StartWorkflowExecution(ctx context.Context, startReq
 	timerTasks := []persistence.Task{&persistence.WorkflowTimeoutTask{
 		VisibilityTimestamp: e.shard.GetTimeSource().Now().Add(timeoutDuration),
 	}}
-	if cronBackoffSeconds != 0 {
+
+	// Only schedule the backoff timer task if not child WF and there's first decision task backoff
+	if cronBackoffSeconds != 0 && startRequest.ParentExecutionInfo == nil {
```

Comment:
Can you also check if we need similar logic on SignalWithStart API?

Reply:
For now, signalWithStart won't check cronSchedule, so no backoff timer task will be scheduled. This is a bug we need to fix. I have created an issue for it. See https://github.com/uber/cadence/issues/1595